### PR TITLE
Fix for zsh

### PR DESCRIPTION
**Pull request by ghost**
Some if's were broken for ZSH

My solution works with **_ZSH 5.1.1**_ and **_BASH 3.2.57(1)**_

### CODE REVIEW

File: getchromium.sh [hunk 1]
```diff
@@ -30,17 +30,17 @@ printf 'Chromium version installed: %s\n' "$INSTALLED_VERSION"
...
 
 # The script should never be run by root
-if [ "$W" = 'root' ]; then
+if [ "$W" == 'root' ]; then
   die 'This script cannot be run as root'
 fi
 
 # Checking if latest available build version number is newer than installed one
-if [ "$LATEST_VERSION" -le "$INSTALLED_VERSION" ]; then
+if (( "$LATEST_VERSION" >= "$INSTALLED_VERSION" )); then
```
Comment (mozamimy) [inline getchromium.sh]: This condition should be,

```bash
(( "$LATEST_VERSION" <= "$INSTALLED_VERSION" ))
```

**Comment by greensn0w** [comment]
@mozamimy Fixed it ;)

**Comment by mozamimy** [comment]
@greensn0w 
Thank you for your contribution :rabbit2: 
I'll merge your modifications!

**Comment by dardo82** [comment]
I have done this that works in zsh to update Chromium:
http://gist.github.com/dardo82/9e815aa00ab4c361e60c

**Comment by ghost** [comment]
@dardo82: Nice idea, but I think we shouldn't force users to install homebrew.
But: We could implement somthing like your solution if homebrew is detected.

I'm currently on a windows machine, so I can't build/test anything right now. 
I'll comment with an exmaple tomorrow :smile: 

BTW: If you and @mozamimy are ok with this, I'd move this discussion to an issue tomorrow, so we can completely close/lock this PR.

**Comment by mozamimy** [comment]
@dardo82 @greensn0w 
Yes, this discussion should move to a new issue.
I'm going to work from now, so i can check @dardo82 's code later :information_desk_person:

**Comment by dardo82** [comment]
OK, I will make a new issue for it...

**Pull request merged**